src/v2/Apps/Partner/Components/PartnerViewingRooms/ViewingRooms.tsx

```diff
@@ -0,0 +1,43 @@
+import { Column, GridColumns, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ViewingRoomCardFragmentContainer } from "./ViewingRoomCard"
+import { ViewingRooms_edges } from "v2/__generated__/ViewingRooms_edges.graphql"
+
+interface ViewingRoomsProps {
+  edges: ViewingRooms_edges
+  eventTitle: string
+}
+
+const ViewingRooms: React.FC<ViewingRoomsProps> = ({ edges, eventTitle }) => (
```

Comment:
We should avoid implicitly returned components and always explicitly use a `return`

Reply:
Okay, thanks! Fixed